scss/_o-step-bar.scss

```diff
@@ -81,27 +83,30 @@
 
 @include media-breakpoint-up(sm) {
   .stepbar-item {
+    position: relative;
     max-width: 3.75rem;
-    padding: map-get($spacers, 2) 0 map-get($spacers, 2) $spacer;
+    padding: $o-stepbar-padding-y 0 $o-stepbar-padding-y $o-stepbar-arrow-width;
     margin: 0;
 
     &::after {
-      display: inline-block;
+      position: absolute;
+      top: 0;
+      right: 1px;
+      z-index: 2;
       width: $o-stepbar-arrow-width;
       height: $o-stepbar-height;
-      margin: -#{map-get($spacers, 2)} 0 -#{map-get($spacers, 2)} auto;
       content: "";
       background-image: escape-svg(url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 17 40'><path fill='#{$black}' d='M0 0l14 20L0 40z'/><path fill='#{$white}' d='M3 0H0l14 20L0 40h3l14-20z'/></svg>"));
       background-size: $o-stepbar-arrow-width $o-stepbar-height;
       transform: translateX(#{"/*rtl:insert:-*/"}$o-stepbar-arrow-width) #{"/*rtl:insert:rotate(180deg)*/"};
     }
 
-    &:last-child::after {
-      display: none;
+    &:first-child {
+      padding-left: divide($o-stepbar-arrow-width, 2);
```

Comment:
Could be a specific variable.

Reply:
It isn't `$o-stepbar-padding-x-xc` rather than `$o-stepbar-paddingx-xc`? And what's `xc`?

Reply:
`xc` stands for `extreme-child`

Reply:
I don't have something better to propose so let's go with that.